### PR TITLE
chore(e2e): Fix e2e tests

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
Peer test PR: https://github.com/superdesk/superdesk/pull/1828

File: spec/send_spec.js [hunk 1]
```diff
@@ -114,8 +106,6 @@ describe('send', function() {
         monitoring.showHideList();
 
         authoring.sendTo('Politic Desk');
-        //Spell check confirmation modal save action
-        authoring.confirmSendTo();
```
Comment (petrjasek) [inline spec/send_spec.js]: it should stay there, problem is the implementation not working with promises..
it's fixed here https://github.com/superdesk/superdesk-client-core/pull/91/files#diff-5c9045fd902f440d5b002c381e24165cR77

Comment (ioanpocol) [inline spec/send_spec.js]: I think that is better to add back on your PR and in this way it will be validated that the issue is fixed.

Comment (petrjasek) [inline spec/send_spec.js]: ok

**Pull request merged**